recipes/libbip3x/bld.bat

```diff
@@ -0,0 +1,59 @@
+@echo off
```

Comment:
For long-term maintainability, it might make sense to re-write this as a single powershell script.

Reply:
Sure, I'll look into it. Could it be simpler to use a single `build.sh`?

Reply:
To use an euphemism, windows is my least favored platform, but let's give the powershell a try, who knows, it may become my salvation

Reply:
Looks good now